pages/_seasons/unit/_unit.vue

```diff
@@ -16,15 +16,31 @@
         <!--        상세 -->
         <div class="main-wrap">
             <div class="unit_top_banner p-relative" :style="`background-image: url(${unitDetail.iconPath})`">
-                <div class="p-absoulte w100p">
+                <div class="p-absoulte w100p over_text">
                     <strong class="text-white">{{ unitDetail.krName }}</strong>
                 </div>
             </div>
+            <div class="unit_content_wrap">
+                <div class="unit_status">
+                    <div class="status_box">
+                        <div class="status_title">체력</div>
+                        <div class="status_value">{{ unitDetail.stats.hp }}</div>
+                    </div>
+                    <div class="status_box">
+                        <div class="status_title">공격력</div>
+                        <div class="status_value">{{ unitDetail.stats.attack }}</div>
+                    </div>
+                </div>
+               <div class="unit_ability_info">
+                   <img class="unit_img_box" :src="unitDetail.ability.iconPath" width="45" height="45" alt="챔피언 특성 이미지" />
```

Comment:
Codacy has a fix for the issue: [Replace `<img·class="unit_img_box"·:src="unitDetail.ability.iconPath"·width="45"·height="45"·alt="챔피언·특성·이미지"` with `·<img⏎························class="unit_img_box"⏎························:src="unitDetail.ability.iconPath"⏎························width="45"⏎························height="45"⏎························alt="챔피언·특성·이미지"⏎···················`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9701485)
```suggestion
                    <img
```